### ISSUE TITLE
Improve task name display 

### ISSUE THREAD

**Issue by kbackowski** [railwaymen/timetable#1]
If we paste https://atlassian.net/projects/XXX/issues/XXX-1114?filter=allissues&board=foo into work time url. Timesheet will display "XXX-1114?filter=allissues&board=foo" as task name. It should display "XXX-1114". We should try to remove all query parameters and take last "/" section of the url.

**Comment by kbackowski** [comment]
We should handle following task urls:

```
https://jira.atlassian.net/browse/XXX-1500
https://www.pivotaltracker.com/story/show/95355474
https://www.meistertask.com/app/task/z8hdvWdR/add-the-option-to-hide-your-profile-if-you-are-a-regular-user
https://trello.com/c/8jr8HV0k/178-replace-google-services-geocoding-maps-by-something-else
https://bm.example.com/projects/66/dashboard/10924
```

**Comment by kbackowski** [comment]
Merged in #6
